Overview.src.html

```diff
@@ -2035,22 +2046,34 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <p>If <var>response</var> is not a
   <span title=concept-network-error>network error</span> and
   <var>response</var> is not a
-  <span title=concept-filtered-response>filtered response</span>, set
-  <var>response</var> to the following
-  <span title=concept-filtered-response>filtered response</span> with
-  <var>response</var> as its
-  <span title=concept-internal-response>internal response</span>, depending on
-  <var>request</var>'s
-  <span title=concept-request-response-tainting>response tainting</span>:
-
-  <dl class="switch compact">
-   <dt>"<code title>basic</code>"
-   <dd><span title=concept-filtered-response-basic>basic filtered response</span>
-   <dt>"<code title>cors</code>"
-   <dd><span title=concept-filtered-response-cors>CORS filtered response</span>
-   <dt>"<code title>opaque</code>"
-   <dd><span title=concept-filtered-response-opaque>opaque filtered response</span>
-  </dl>
+  <span title=concept-filtered-response>filtered response</span>, run these substeps:
+
+  <ol>
+   <li>
+    <p>Set <var>response</var>'s
+    <span title=concept-response-cors-exposed-header-name-list>CORS-exposed header-name list</span>
+    to the result of <span title=concept-header-parse>parsing</span>
+    `<code title=http-access-control-expose-headers>Access-Control-Expose-Headers</code>` in
+    <var>response</var>'s
+    <span title=concept-response-header-list>header list</span>.
```

Comment:
Although it does not matter in a black-box observable way, I still think we should add a conditional here for response tainting being "cors".